NETDefault/WebApp/Program.cs

```diff
@@ -7,10 +7,14 @@ namespace WebApp
 {
     public class Program
     {
+        public static string Instance = Guid.NewGuid().ToString();
```

Comment:
A disadvantage of this is that the instance name resets after a restart..

Reply:
Yes, this is to make it clear that the instance changed (after swap etc), and to see which instance of the load-balancing pool has served you content.

Reply:
Added comment to make sure this is only for demonstrating the concept.